src/index.js

```diff
@@ -132,7 +132,7 @@ export default function supernova(env) {
         }
       }, [expandedState, containerData, selectionObj.state]);
 
-      snapshot(expandedState, containerData, layout, transform, initialZoom);
+      snapshot(expandedState, containerData, layout, transform, initialZoom, element);
```

Comment:
Why are we passing it in here. In the function we have useElement and don't get it from the parameters

Reply:
SHould not be there good catch